tasks/unit-tests/libs/common/git_tests.py

```diff
@@ -15,3 +15,10 @@ def test_get_staged_files(self):
 
         self.assertEqual(files, ["file1", "file2", "file3"])
         self.ctx_mock.run.assert_called_once_with("git diff --name-only --staged HEAD", hide=True)
+
+    def test_get_current_branch(self):
+        self.ctx_mock.run.return_value.stdout = "  main  \n"
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Smart